File: Sum_function.py
def summing_arguments(numbers):
    sum_of_args = 0
    for i in numbers:
        sum_of_args += int(i)
    return sum_of_args

print("Please, enter some numbers separated with commas and spaces (e.g. '13, 15, 17, 23')")
user_input = input()
modified_user_input = user_input.split(", ")
print(summing_arguments(modified_user_input))

File: Sum_function.js
function summingArguments(numbers) {
    let sumOfArgs = 0;
    for (let i of numbers) {
        sumOfArgs += parseInt(i); // converting a string argument (i) to an integer to sum arguments properly
    }
    return sumOfArgs;
}

let _userInput = prompt("Please, enter some numbers separated with commas&spaces (e.g. '13, 15, 17, 23')");
let $modifiedUserInput = _userInput.split(", ");
console.log(summingArguments($modifiedUserInput));
